readme.txt

```diff
@@ -1,10 +1,8 @@
 === Autopost for X (formerly Autoshare for Twitter) ===
 Contributors:      10up, johnwatkins0, adamsilverstein, scottlee, dinhtungdu, jeffpaul, dharm1025
-Tags:              twitter, tweet, autoshare, auto-share, auto share, share, sharing, social media, posse, x, x/twitter, twitter/x
-Requires at least: 5.7
+Tags:              twitter, tweet, share, social media, posse
```

Comment:
I know this isn't a new change here but wondering why we have `posse` as a tag for this?

Reply:
It's an indieweb acronym: https://indieweb.org/POSSE